tests/unit/amo/sagas/test_users.js

```diff
@@ -244,7 +244,7 @@ describe(__filename, () => {
         .once()
         .returns(Promise.resolve(allNotifications));
 
-      sagaTester.dispatch(editUserAccount({
+      sagaTester.dispatch(updateUserAccount({
         errorHandlerId: errorHandler.id,
         notifications,
         picture: null,
```

Comment:
There's a line below (line 266), on which the description should be changed from `it('cancels the edit and dispatches...` to `it('cancels the update and dispatches ...`.